tests/test_database_manager.py

```diff
@@ -42,54 +42,53 @@ class TestStuff(object):
     manager = DatabaseManager()

```

Comment:
This will cause a collection failure when the Db is not available, completely breaking the test process.

Instead of creating the instance during **class load** this should be done in a fixture